src/component/Pagination.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Pagination, Stack } from "@mui/material";
+
+const itemsPerPage = 12;
+
+const PaginationComponent = ({ totalItems, currentPage, onPageChange }) => {
```

Comment:
you should never called the component SomethingComponent
you should give it a name with linking it with Component
so this must be 

```suggestion
const Pagination = ({ totalItems, currentPage, onPageChange }) => {
```